pkgs/applications/misc/resp-app/default.nix

```
@@ -0,0 +1,116 @@
+{ stdenv
+, mkDerivation
+, lib
+, fetchFromGitHub
+, brotli
+, lz4
+, pyotherside
+, python3
+, python3Packages
+, qtbase
+, qtcharts
+, qtdeclarative
+, qtgraphicaleffects
+, qtimageformats
+, qmake
+, qtquick1
+, qtquickcontrols2
+, qtsvg
+, qttools
+, snappy
+, wrapQtAppsHook
+, zstd
+}:
+
+let
+  redis-rdb-tools = with python3Packages; buildPythonPackage rec {
+    pname = "redis-rdb-tools";
+    version = "resp_rdbtools-0.1.15";
+
+    src = fetchFromGitHub {
+      owner = "uglide";
+      repo = "redis-rdb-tools";
+      rev = version;
+      sha256 = "sha256-uoDNJ0Yzyf/4VM1n6e2rsy7/IdCJNdnD/HcqTRCkjwY=";
+    };
+
+    propagatedBuildInputs = [
+      redis
+    ];
+
+    doCheck = false;
+  };
+in
+mkDerivation rec {
+  pname = "RESP.app";
+  version = "2022.5";
+
+  src = fetchFromGitHub {
+    owner = "RedisInsight";
+    repo = "RedisDesktopManager";
+    fetchSubmodules = true;
+    rev = version;
+    sha256 = "sha256-5eI3J2RsYE5Ejb1r8YkgzmGX2FyaCLFD0lc10J+fOT4=";
+  };
+
+  nativeBuildInputs = [
+    python3Packages.wrapPython
+    qmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    brotli lz4 pyotherside python3 qtbase qtdeclarative qtgraphicaleffects
+    qtimageformats qtquick1 qtquickcontrols2 qtsvg qttools qtcharts snappy zstd
+  ] ++ pythonPath;
+
+
+  pythonPath = with python3Packages; [
+    bitstring
+    cbor
+    msgpack
+    phpserialize
+    redis-rdb-tools
+    python-lzf
+  ];
+
+  patchPhase = ''
+    substituteInPlace src/resp.pro \
+      --replace 'which ccache' "false"
+    substituteInPlace src/resp.pro \
+      --replace 'target.files = $$DESTDIR/resp' "${placeholder "src"}/bin/linux/release/resp"
+    substituteInPlace src/resp.pro \
+      --replace '/opt/resp_app' "${placeholder "out"}"
+    substituteInPlace src/resp.pro \
+      --replace 'target.path = $$LINUX_INSTALL_PATH' 'target.path = $$LINUX_INSTALL_PATH/bin'
+    substituteInPlace src/resp.pro \
+      --replace '/usr/' "$out/"
+  '';
+
+  qmakeFlags = [
+    "SYSTEM_LZ4=1"
+    "SYSTEM_ZSTD=1"
+    "SYSTEM_SNAPPY=1"
+    "SYSTEM_BROTLI=1"
+    "VERSION=${version}"
+    "src/resp.pro"
+  ];
+
+  dontWrapQtApps = true;
+
+  postInstall = ''
+    buildPythonPath "$out $pythonPath"
+    wrapProgram $out/bin/resp \
+      --prefix PYTHONPATH : "$PYTHONPATH" \
+      "''${qtWrapperArgs[@]}"
+  '';
+
+  meta = with lib; {
+    broken = (stdenv.isLinux && stdenv.isAarch64);
+    description = "Cross-platform Developer GUI for Redis";
+    homepage = "https://resp.app/";
+    license = licenses.gpl3Only;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ cstrahan ];
```

```suggestion
    description = "Cross-platform Developer GUI for Redis";
    homepage = "https://resp.app/";
    license = licenses.gpl3Only;
    platforms = platforms.linux;
    maintainers = with maintainers; [ cstrahan ];
    broken = stdenv.isLinux && stdenv.isAarch64;
```
Is it still broken on aarch64-linux?

---

Let's find it out :smile: